code/modules/tgui_panel/stat.dm

```diff
@@ -112,3 +114,26 @@
 	if(!is_ready())
 		return
 	window.send_message("stat/clearAlertPopup", list())
+
+/**
+ * public
+ *
+ * Displays the message asking an admin to start battle royale
+ */
+/datum/tgui_panel/proc/give_br_popup(title, text)
+	if(!is_ready())
+		return
+	var/list/payload = list()
+	payload["title"] = title
+	payload["text"] = text
```

Comment:
I wouldnt be against hardcoding this, since it never changes and removes some data sent to TGUI